### DISCUSSION TITLE
Save an element in Craft CMS without creating a revision

### DISCUSSION THREAD

**Discussion by cmartinespinosa** [craftcms/cms#13075] [Q&A]
Is it possible to save an element in Craft CMS without creating a revision? Sometimes I don't want to create a revision based on the parameters. Here's an example of my code:

```
$project->setFieldValue('chartImageDetails', [$asset->id]);
Craft::$app->elements->saveElement($project); // i dont want to create a revision 
```

Thanks

**Comment by brandonkelly** [comment]
Little hacky, but entries will only create new revisions when they aren’t being “resaved”, so you could pretend that’s the cased to opt out of the revision creation:

```php
$project->setFieldValue('chartImageDetails', [$asset->id]);
$project->resaving = true;
Craft::$app->elements->saveElement($project);
```
